common/djangoapps/django_comment_common/models.py

```diff
@@ -105,3 +106,29 @@ class Meta:
 
     def __unicode__(self):
         return self.name
+
+
+def all_permissions_for_user_in_course(user, course_id):  # pylint: disable=invalid-name
+    """Returns all the permissions the user has in the given course."""
+    course = modulestore().get_course(course_id)
+    if course is None:
+        raise ItemNotFoundError(course_id)
+    course_blacked_out = not course.forum_posts_allowed
+
+    all_roles = {role.name for role in Role.objects.filter(users=user, course_id=course_id)}
+
+    def permission_blacked_out(p_name):
+        """Returns true if the given permission is blacked out"""
```

Comment:
Nit: I'm not sure what it means for a permission to be "blacked out". Can the comment be more helpful (instead of just repeating the name of the method)?